package.json

```diff
@@ -51,12 +51,9 @@
 		"test:framework-plugins": "lerna run test --scope=@liskhq/lisk-framework-*",
 		"test:commander": "lerna run test --scope=lisk-commander",
 		"build": "lerna run build",
-		"init": "./scripts/init.sh"
-	},
-	"husky": {
-		"hooks": {
-			"pre-commit": "lint-staged"
-		}
+		"init": "./scripts/init.sh",
+		"prepare": "husky install",
+		"postinstall": "husky install"
```

Comment:
`prepare` and `postinstall` both runs for me on yarn1. probably only `postintall` is fine?

Reply:
> Yarn 2 doesn't support `prepare`  lifecycle script, so husky needs to be installed differently (this doesn't apply to Yarn 1 though). See Yarn 2 install.

https://typicode.github.io/husky/#/

That's from their docs, I thought to add both, so when in future we upgrade `yarn` we don't miss this. Let me know if you think to remove one.